mautic/mautic
# Fix integration special fields order

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:



#### Steps to test this PR:
Noticed special fields like contact ID, timeline link and do not contact by email in integration are not correct in field list.  

![image](https://user-images.githubusercontent.com/462477/143567098-d1508e48-d98c-4096-bec0-eb02438e9ed0.png)



1. Load up [this PR](https://mautibox.com)
2. Setup any integration
3. Try search mautic in Mautic field
You should see 

![image](https://user-images.githubusercontent.com/462477/143567098-d1508e48-d98c-4096-bec0-eb02438e9ed0.png)

4.  After patch you see correct fields in mapping
![image](https://user-images.githubusercontent.com/462477/143567329-ead0daf2-3733-45df-a43a-c64642f55c29.png)





<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/10649"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
